frontend/src/layout/RTCLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback, useEffect } from "react";
+import { Outlet } from "react-router-dom";
+import { useRecoilValueLoadable, useSetRecoilState } from "recoil";
+import { ensembleAtom, userListAtom } from "../recoil/ensemble";
+import { socketAtom } from "../recoil/socket";
+import { Ensemble, EnsembleObject } from "common/dist";
+
+const RTCLayout: React.FC = () => {
```

Comment:
I'd like you to add a JSDoc docstring to this component. Add a note about the responsibility of this component. It can be a few sentences long to make sure we remember this in like a month.